ml

```diff
@@ -204,106 +199,95 @@ def init(
         )
         return
 
-    prompt = """
-    WARNING: You have requested to initialize Mole.
-    This will permanently overwrite your current database. Do you wish to proceed?
-
-    Do you wish to continue? [y/N]: """
-
-    sys.stdout.write(prompt)
-    if sys.version_info.major == 3:
-        choice = input().lower()
-    else:
-        choice = raw_input().lower()
-
-    if choice in yes:
+    cmd = [
+        "docker-compose",
+        "up",
+        "--build",
+        "--force-recreate",
+        "--always-recreate-deps",
+    ]
+    if os.environ.get("UNLOCK_REDIS") or unlock_redis:
         cmd = [
             "docker-compose",
+            "-f",
+            "docker-compose.yml",
+            "-f",
+            "docker-compose-unlocked-redis.yml",
             "up",
             "--build",
             "--force-recreate",
             "--always-recreate-deps",
         ]
-        if os.environ.get("UNLOCK_REDIS") or unlock_redis:
-            cmd = [
-                "docker-compose",
-                "-f",
-                "docker-compose.yml",
-                "-f",
-                "docker-compose-unlocked-redis.yml",
-                "up",
-                "--build",
-                "--force-recreate",
-                "--always-recreate-deps",
-            ]
-        # Don't run portainer if "lite" flag is set
-        if lite:
-            SERVICES.discard("portainer")
-            SERVICES.discard("docs")
-            # Should reverse-proxy always be included even under lite mode?
-            SERVICES.discard("proxy")
-            SERVICES.discard("event_generator")
-            SERVICES.discard("zookeeper1")
-            SERVICES.discard("zookeeper2")
-            SERVICES.discard("zookeeper3")
-            SERVICES.discard("bookkeeper1")
-            SERVICES.discard("bookkeeper2")
-            SERVICES.discard("bookkeeper3")
-            SERVICES.discard("pulsar")
-            SERVICES.discard("pulsar_proxy")
-        if not db_backup:
-            SERVICES.discard("db_backup")
-        if not angular or skip_static_build:
-            SERVICES.discard("angular")
-
-        if make_migrations:
-            MAKE_MIGRATIONS_FLAG = "true"
-        else:
-            MAKE_MIGRATIONS_FLAG = "false"
-
-        # Don't run map tile server if there are no tile sets available
-        available_mbtiles = glob.glob(MAP_TILES_PATH + "*.mbtiles")
+    # Don't run portainer if "lite" flag is set
+    if lite:
```

Comment:
think we can remove zookeepers, bookkeepers, and pulsar proxy as these services don't exist anymore